playground/seq/Seq.ServiceDefaults/Extensions.cs

```diff
@@ -46,8 +46,9 @@ public static IHostApplicationBuilder ConfigureOpenTelemetry(this IHostApplicati
         builder.Services.AddOpenTelemetry()
             .WithMetrics(metrics =>
             {
-                metrics.AddRuntimeInstrumentation()
-                       .AddBuiltInMeters();
+                metrics.AddAspNetCoreInstrumentation()
```

Comment:
Any chance that while you are editing these files, can you sync them up to the latest template content? There are a couple other diffs.

Not saying do it for all the Extensions.cs files (but I won't stop you from doing it 😉) I just thought that since we are modifying these files anyway....

Reply:
Sure - can you point me at the bit you mean just so I get the right thing and I'll do that shortly.

Reply:
Ah, sorry. I should have linked.

https://github.com/dotnet/aspire/tree/main/src/Aspire.ProjectTemplates/templates/aspire-servicedefaults

is the "source of truth" for what the ServiceDefaults template is.

Reply:
I _think_ I've got them all - I just copied the edits from #2864.